ONSdigital/eq-survey-runner
# eq-908 Corrupted ONS logo

### What is the context of this PR?

Fixes #908 by removing the svgmin process from gulp task that copies the file to static

### How to review 

1. remove all static assets
2. `yarn compile`
3. check outputted logo.svg is uncompressed and valid svg
